<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IN3261049W1" failures="1" tests="3" name="Test" time="15.184" errors="0" timestamp="2022-08-12T16:12:35 IST">
  <testcase classname="com.learnautomation.testcases.AddUserTest" name="loginApp" time="2.57">
    <failure type="java.lang.AssertionError" message="Login failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed expected [true] but found [false]
at com.learnautomation.testcases.AddUserTest.loginApp(AddUserTest.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginApp -->
  <testcase classname="com.learnautomation.testcases.AddUserTest" name="addUserDetails" time="0.0">
    <skipped/>
  </testcase> <!-- addUserDetails -->
  <testcase classname="com.learnautomation.testcases.AddUserTest" name="logoutTest" time="0.0">
    <skipped/>
  </testcase> <!-- logoutTest -->
</testsuite> <!-- Test -->
